ajax/libs/leaflet-polylinedecorator/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "leaflet-polylinedecorator",
+  "filename": "leaflet.polylineDecorator.min.js",
+  "npmName": "leaflet-polylinedecorator",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "leaflet-polylinedecorator*"
+      ]
+    }
+  ],
+  "version": "1.0.1",
+  "authors": [
+    "Benjamin",
+    "Becquet"
```

Comment:
Did the bower.json say 
```js
"authors": [
    "Benjamin",
    "Becquet"
]
```

I don't even see it, you should double check your proof ...
https://github.com/bbecquet/Leaflet.PolylineDecorator/blob/master/bower.json#L6